contrib/inventory/azure_rm.py

```diff
@@ -812,6 +897,12 @@ def _load_settings(self):
                 except:
                     pass
 
+            if not config.has_section('cache'):
+                config.add_section('cache')
+
+            settings['cache_path'] = config.get('cache', 'cache_path')
+            settings['cache_max_age'] = config.getint('cache', 'cache_max_age')
+            settings['cache_filename'] = config.get('cache', 'cache_filename')
```

Comment:
For this `cache` section, why don't you use a similar dict as `AZURE_CONFIG_SETTINGS `?

Reply:
I need to write a lot of code in a couple of places to accommodate that and tbh I don't the env vars are doing their job, they should overwrite what is read from a file and they are not doing that :(

Reply:
pls add those in environment variables for consistent configuration behavior